### ISSUE TITLE
Documentation: White text on color swatches is not accessible

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-react#2017]
### Bug Report

- __Package version(s)__: Latest website
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Maybe

Requested priority: Normal

Products/sites affected: Fabric website

#### Describe the issue:

The text on the theme and neutral color tables does not meet the minimum contrast requirement of 4.5:1.

![image](https://user-images.githubusercontent.com/1382445/27297584-4942fa8e-54da-11e7-8ad9-42fafb6e4e09.png)
